easysocial/easysocial/comments.php

```diff
@@ -159,13 +167,25 @@ public function getComments()
 		$mapp				=	new EasySocialApiMappingHelper;
 		$data				=	$mapp->createCommentsObj($row, $row->limitstart, $row->limit);
 
-		if (count($data['data']) < 1)
+		// Determine if this user has the permissions to read comment.
+		$access 	= FD::access();
+		$allowed	= $access->get('comments.read');
+
+		if (!$allowed)
 		{
-			$res->empty_message	=	JText::_('APP_USER_KOMENTO_NO_COMMENTS_FOUND');
+			$res->empty_message	=	JText::_('PLG_API_EASYSOCIAL_READ_COMMENT_NOT_ALLOW_MESSAGE');
+			$res->status		=	false;
```

Comment:
@Pratikshakhandagale  - Use ApiError::raiseError method with proper status code (403) for unauthorised call.